app/core/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+
+class Recipe(models.Model):
+    """Recipe object"""
+    name = models.CharField(max_length=255)
+    description = models.CharField(max_length=255)
+
+    def create(self, validated_data):
+        ingredients = validated_data.pop('ingredients')
+        recipe = Recipe.objects.create(**validated_data)
+        if ingredients:
+            for ingredient in ingredients:
+                Ingredient.objects.create(recipe=recipe, **ingredient)
+        return recipe
```

Comment:
Better make this a classmethod (or a staticmethod). Because you shouldn't have the `self` when you try to create a new instance.
```suggestion
    @classmethod
    def create(cls, validated_data):
        ingredients = validated_data.pop('ingredients')
        recipe = cls.objects.create(**validated_data)
        if ingredients:
            for ingredient in ingredients:
                Ingredient.objects.create(recipe=recipe, **ingredient)
        return recipe
```

Reply:
I'm removing this `create` method from the model, as it is not needed here anyway (it is already on the serializer).